<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>defoltwindow</class>
 <widget class="QDialog" name="defoltwindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>дефолтный калькулятор</string>
  </property>
  <widget class="QPushButton" name="Back">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>486</y>
     <width>540</width>
     <height>109</height>
    </rect>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num0">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSqrt">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>sqrt</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonMod">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>mod</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="shortcut">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>sin</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mult">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
   <property name="shortcut">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAtan">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>atan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_LeftBracket">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>(</string>
   </property>
   <property name="shortcut">
    <string>(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonTan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>tan</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAsin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>asin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="result">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPoint">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="shortcut">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAcos">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>acos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="division">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="shortcut">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLog">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>log</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPow">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>pow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCos">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>cos</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>ln</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_RightBracket">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>)</string>
   </property>
   <property name="shortcut">
    <string>)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_number">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="minus">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="graf">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>-10</y>
     <width>90</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string>график</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="result_x">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999999999999999983222784.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000000000004764729344.000000000000000</double>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>360</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser {

	background-color: rgba(255, 34, 251, 128);
  color: white;
  border: 1px solid gray;
}</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;x =&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:24pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;начало координат x=                            y=&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;конец  координат  x=                            y=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_X">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>361</y>
     <width>90</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
   </property>
   <property name="text">
    <string>Х</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="result_x_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="result_x_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="result_x_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="result_x_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>460</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
  <zorder>textBrowser</zorder>
  <zorder>Back</zorder>
  <zorder>num0</zorder>
  <zorder>ButtonSqrt</zorder>
  <zorder>num7</zorder>
  <zorder>num3</zorder>
  <zorder>plus</zorder>
  <zorder>ButtonMod</zorder>
  <zorder>num9</zorder>
  <zorder>ButtonSin</zorder>
  <zorder>mult</zorder>
  <zorder>ButtonAtan</zorder>
  <zorder>num2</zorder>
  <zorder>Button_LeftBracket</zorder>
  <zorder>ButtonTan</zorder>
  <zorder>ButtonAsin</zorder>
  <zorder>result</zorder>
  <zorder>ButtonPoint</zorder>
  <zorder>num8</zorder>
  <zorder>ButtonAcos</zorder>
  <zorder>division</zorder>
  <zorder>ButtonLog</zorder>
  <zorder>ButtonPow</zorder>
  <zorder>ButtonCos</zorder>
  <zorder>ButtonLn</zorder>
  <zorder>num6</zorder>
  <zorder>delete_2</zorder>
  <zorder>num4</zorder>
  <zorder>num5</zorder>
  <zorder>Button_RightBracket</zorder>
  <zorder>num1</zorder>
  <zorder>result_number</zorder>
  <zorder>minus</zorder>
  <zorder>graf</zorder>
  <zorder>result_x</zorder>
  <zorder>but_X</zorder>
  <zorder>result_x_2</zorder>
  <zorder>result_x_3</zorder>
  <zorder>result_x_4</zorder>
  <zorder>result_x_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
